x-pack/plugins/observability/public/components/shared/page_template/nav_name_with_beta_badge.tsx

```diff
@@ -4,45 +4,44 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-import { EuiBetaBadge } from '@elastic/eui';
-import type { IconType } from '@elastic/eui/src/components/icon/icon';
-import { i18n } from '@kbn/i18n';
 import React from 'react';
-import styled from 'styled-components';
+import { i18n } from '@kbn/i18n';
+import { EuiBetaBadge, EuiFlexGroup, EuiFlexItem, IconType } from '@elastic/eui';
 
 interface Props {
-  iconType: IconType;
-  label: string;
+  label?: string;
+  isTechnicalPreview?: boolean;
+  iconType?: IconType;
 }
 
-const LabelContainer = styled.span`
-  max-width: 72%;
-  float: left;
-  &:hover,
-  &:focus-within,
-  &:focus {
-    text-decoration: underline;
-  }
-`;
-
-const StyledBetaBadge = styled(EuiBetaBadge)`
-  margin-left: 5px;
-  margin-bottom: -6px;
-`;
-
-export function NavNameWithBetaBadge({ label, iconType }: Props) {
+export function NavNameWithBetaBadge({ label, iconType, isTechnicalPreview }: Props) {
   return (
-    <>
-      <LabelContainer className="eui-textTruncate">
-        <span>{label}</span>
-      </LabelContainer>
-      <StyledBetaBadge
-        label={i18n.translate('xpack.observability.navigation.experimentalBadgeLabel', {
-          defaultMessage: 'Technical preview',
-        })}
-        size="s"
-        iconType={iconType}
-      />
-    </>
+    <EuiFlexGroup alignItems="center" gutterSize="s">
+      <EuiFlexItem grow={false}>
+        <span className="eui-textTruncate">
+          <span>{label}</span>
+        </span>
+      </EuiFlexItem>
+      <EuiFlexItem grow={false} style={{ height: 20 }}>
+        {isTechnicalPreview ? (
+          <EuiBetaBadge
+            color={'hollow'}
```

Comment:
nit: I missed that in my comment - you can change it to 
```suggestion
            color="hollow"
```